mmcls/models/backbones/efficientnet_v2.py

```diff
@@ -0,0 +1,330 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import math
+from functools import partial
+from typing import Callable, List, Optional, Tuple
+
+import torch
+import torch.nn as nn
+import torch.utils.checkpoint as cp
+from mmcv.cnn.bricks import DropPath, ConvModule
+from mmengine.model import BaseModule, Sequential
+from torch import Tensor
+
+from mmcls.models.backbones.base_backbone import BaseBackbone
+from mmcls.models.utils import InvertedResidual, SELayer
+from mmcls.registry import MODELS
+
+
+# copy from efficientnetv1
+class EdgeResidual(BaseModule):
```

Comment:
if is the same with `mmcls.models.backbones.efficientv1.EdgeResidual`, why not just import from it?
Or inherit one from it and then the new class name is MBConv as well as FusedMBConv, like in the paper.

Reply:
>  (EdgeResidual is like MBConv , InvertedResidual is FusedMBConv).

it would be better to add comments about this information to help the users to understand the code.